homeassistant/components/tesla/__init__.py

```diff
@@ -134,7 +134,8 @@ async def async_setup_entry(hass, config_entry):
     """Set up Tesla as config entry."""
     hass.data.setdefault(DOMAIN, {})
     config = config_entry.data
-    websession = aiohttp_client.async_get_clientsession(hass)
+    # Because users can have multiple accounts, we always create a new session
```

Comment:
This comment doesn't explain why it is needed. You said something about cookies.

Reply:
Ok.